docs/device-agent/install.md

```diff
@@ -22,6 +22,7 @@ The Device Agent connects back to the FlowFuse platform on port 443. You will ne
  - `app.flowfuse.com:443`
  - `mqtt.flowfuse.cloud:443`
 
+Additionally, ensure your firewall permits access to the npm registry. The Device Agent does not install Node-RED itself; when it receives a snapshot to run, it will need to download the required Node-RED version specified in that snapshot.
```

Comment:
@hardillb do you happen to know what endpoint that would be?

Reply:
`registry.npmjs.com`

```suggestion
Additionally, ensure your firewall permits access to the npm registry. The Device Agent does not install Node-RED at startup; but when it receives a snapshot to run, it will need to download the required Node-RED version specified in that snapshot.
```